### PR TITLE
[doc] akka-scala and Java README normalize

### PR DESCRIPTION
**Pull request by shijinkui**
akka-scala readme have some arbitrary place. Make Java and Akka-scala readme in the same style.

In the SDK readme should not have swagger version info, or that will confuse the endpoint user.

issue: #7380

ping @jimschubert  @bbdouglas

### CODE REVIEW

**Comment by shijinkui** [comment]
@wing328 generated sdk readme is fail to our standard, need fix https://github.com/huaweicloudsdk/huaweicloud-cs-sdk/tree/master/huaweicloud-cs-scala-sdk

File: modules/swagger-codegen/src/main/resources/akka-scala/README.mustache [hunk 1]
```diff
@@ -1,53 +1,41 @@
...
-- Build date: {{generatedDate}}
-{{/hideGenerationTimestamp}}
-- Build package: {{generatorClass}}
-{{#infoUrl}}
-For more information, please visit [{{{infoUrl}}}]({{{infoUrl}}})
-{{/infoUrl}}
-
-# Requirements
-
-Building the API client library requires [Maven](https://maven.apache.org/) to be installed.
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/akka-scala/README.mustache]: Understood that you don't need these section in the auto-generated README but can we still keep this while you maintain a customized version of the templates to meet your unique requirement (e.g. putting Huawei logo in the README)?

This part is pretty standard across all the auto-generated README.

Comment (shijinkui) [reply modules/swagger-codegen/src/main/resources/akka-scala/README.mustache]: Not only need maven but also need jdk and other `Requirements `.
Here I add a list of tools.

Comment (wing328) [reply modules/swagger-codegen/src/main/resources/akka-scala/README.mustache]: Sorry I wasn't clear. I'm referring to line 9 to 21

Comment (shijinkui) [reply modules/swagger-codegen/src/main/resources/akka-scala/README.mustache]: `Automatically generated by the ...` such doc not exist in the Java readme.  `Build date` and `API version` can be kept. We keep the same with Java readme.

Comment (shijinkui) [reply modules/swagger-codegen/src/main/resources/akka-scala/README.mustache]: fixed. move brief version info to project section.

**Pull request merged**